configuration/configuration.go

```diff
@@ -350,6 +350,8 @@ func (storage Storage) Type() string {
 			// allow configuration of caching
 		case "delete":
 			// allow configuration of delete
+		case "foreignlayers":
```

Comment:
I see why this is a logical place. In the past, we had `Compatibility` to include manifest validation, which was top-level.